icon_validator/rules/plugin_validators/help_input_output_validator.py

```diff
@@ -83,7 +83,7 @@ def get_spec_input(input_content: dict) -> list:
             required = input_content.get(k).get("required")
             description = input_content.get(k).get("description")
             enum = input_content.get(k).get("enum", None)
-            example = input_content.get(k).get("example", None)
```

Comment:
What was the reasoning behind removing the default of None here?

Reply:
I want to make it so you have to have added an example.

Reply:
Ah right, that makes sense

Reply:
`None` is also the default argument by default for `.get()` :)